src/main/kotlin/gg/skytils/skytilsmod/features/impl/misc/SlayerFeatures.kt

```diff
@@ -1387,4 +1388,18 @@ object SlayerFeatures : CoroutineScope {
             }
         }
     }
+
+    class VampireSlayer(entity: EntityOtherPlayerMP) :
```

Comment:
Should be called BloodfiendSlayer to match the other 2

Reply:
should be RiftstalkerSlayer because we take the first part of the name

Reply:
Hypixel calls it bloodfiend, inferno demon lord is demonlordslayer

Reply:
ok